packages/block-library/src/gallery/index.php

```diff
@@ -56,8 +56,10 @@ function block_core_gallery_render( $attributes, $content ) {
 		$content,
 		1
 	);
-	$gap_value = $gap ? $gap : 'var( --wp--style--block-gap, 0.5em )';
-	$style     = '.' . $class . '{ --wp--style--unstable-gallery-gap: ' . $gap_value . '}';
+	// --gallery-block--gutter-size is deprecated. --wp--style--gallery-gap-default should be used by themes that want to set a default
+	// gap on the gallery.
+	$gap_value = $gap ? $gap : 'var( --gallery-block--gutter-size, var( --wp--style--gallery-gap-default, var( --wp--style--block-gap, 0.5em ) ) )';
```

Comment:
Totally optional and it probably doesn't really matter, but for this rule, since `--gallery-block--gutter-size` is the deprecated variable, would it be better to have `--wp--style--gallery-gap-default` take precedence?

Reply:
yeh, good point, done.